react/src/lib/EventOverlay/index.js

```diff
@@ -686,15 +686,14 @@ class EventOverlay extends React.Component {
         break;
       case 'left':
         if (!scrollParentDims.left && !transformParentDims) {
-          if (
-            arrowWidth + offsetWidth + elementDims.width + anchorPosition.left >
-            anchorPosition.left
-          ) {
+          targetNode.style.right = `${documentRight -
+            anchorPosition.left +
+            arrowWidth +
+            offsetWidth}px`;
```

Comment:
We always need to specify the right hand position here. the previous if check was pretty much always true so for the majority of cases the logic hasn't changed.